build/dependencies.props

```diff
@@ -1,16 +1,25 @@
 <Project>
   <PropertyGroup>
-    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+    <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <CoreFxVersion>4.3.0</CoreFxVersion>
+    <DapperVersion>1.*</DapperVersion>
+    <InternalAspNetCoreSdkVersion>2.0.0-*</InternalAspNetCoreSdkVersion>
+    <JsonNetVersion>9.0.1</JsonNetVersion>
```

Comment:
Why are we using a lower Json.NET version here than in any other repo? Should be `10.0.1`.